client/build.bat

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
```

Comment:
I think it is supposed to be a Windows script because of `rmdir /S /Q`, and that is Windows syntax.

If it is a Windows script, please use `.cmd` extension (.bat is older, and does not have exactly the same behavior, as it handles returning errorlevels differently). 

Also, if it is for Windows, I expect it to work without having bash installed. Please make sure this runs also in cmd.exe or powershell at least.

Reply:
Good point, I have updated the script and package.json accordingly.